### DISCUSSION TITLE
Strange characters in output on Windows Server 2012R2

### DISCUSSION THREAD

**Discussion by garyaderhold** [johnkerl/miller#1644] [General]
Any idea why the output would show these strange characters?

C:\miller-6.12.0-windows-386>mlr --csv cat example.csv
←[1m←[4mcolor←[0m,←[1m←[4mshape←[0m,←[1m←[4mflag←[0m,←[1m←[4mk←[0m,←[1m←[4mindex←[0m,←[1m←[4mquantity←[0m,←[1m←[4mrate←[0m
yellow←[0m,triangle←[0m,true←[0m,1←[0m,11←[0m,43.6498←[0m,9.8870←[0m
red←[0m,square←[0m,true←[0m,2←[0m,15←[0m,79.2778←[0m,0.0130←[0m
red←[0m,circle←[0m,true←[0m,3←[0m,16←[0m,13.8103←[0m,2.9010←[0m
red←[0m,square←[0m,false←[0m,4←[0m,48←[0m,77.5542←[0m,7.4670←[0m
purple←[0m,triangle←[0m,false←[0m,5←[0m,51←[0m,81.2290←[0m,8.5910←[0m
red←[0m,square←[0m,false←[0m,6←[0m,64←[0m,77.1991←[0m,9.5310←[0m
purple←[0m,triangle←[0m,false←[0m,7←[0m,65←[0m,80.1405←[0m,5.8240←[0m
yellow←[0m,circle←[0m,true←[0m,8←[0m,73←[0m,63.9785←[0m,4.2370←[0m
yellow←[0m,circle←[0m,true←[0m,9←[0m,87←[0m,63.5058←[0m,8.3350←[0m
purple←[0m,square←[0m,false←[0m,10←[0m,91←[0m,72.3735←[0m,8.2430←[0m

C:\miller-6.12.0-windows-386>mlr --version
mlr 6.12.0

**Comment by johnkerl** [comment]
Yes this is due to attempts to colorize output. Which I try avoid if "on Windows" ... and apparently that's being misdetected.

Can you try `set MLR_NO_COLOR=true` at your prompt? That will help diagnose. Also I'll find the is-windows logic & try to see why it's not triggering here ...

**Comment by garyaderhold** [comment]
"set MLR_NO_COLOR=true" fixed it.  Thanks.

**Comment by johnkerl** [comment]
Awesome! :)
